apps/homework/templates/homework/overview.html

```diff
@@ -7,14 +7,24 @@
 
 {% block extra_head %}
     <style>
-        .condensed {
-            margin: 0.5vh 0 0 0 !important;
-            padding: 0.5vh 0 0 0 !important;
+        div.ui.statistic {
+            padding: 1em 0 0 0;
+            width: 100%;
         }
 
```

Comment:
See here, there's one line separating this css class from the one following it. We should have uniform formatting for all of these css classes, and I'd like to see a new line between each of them.

Reply:
Gotcha. Fixed now.

Reply:
Gotcha. Fixed now